docs/team/eugenetangkj.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Eugene Tang's Project Portfolio Page
+---
+
+## About InternBuddy
+
+InternBuddy is a one-stop and simple platform for **Computing undergraduates** to manage and keep track of their
+internship applications. It is a desktop application that is optimised for **fast typists** where interactions
+primarily revolve around text input, although it does provide a graphical user interface for users.
+
+Implementation-wise, InternBuddy is created using **Java 11**, with usage of the [JavaFX](https://openjfx.io/),
+[Jackson](https://github.com/FasterXML/jackson) and [JUnit5](https://github.com/junit-team/junit5) libraries. It
+is adapted from the AddressBook-Level3 project created by the [SE-EDU initiative](https://se-education.org).
```

Comment:
I don't think it is necessary to put implementation details in the portfolio page. Maybe it should be placed in the developer's guide?